src/Features/CSharp/Portable/BraceCompletion/LessAndGreaterThanBraceCompletionService.cs

```diff
@@ -73,12 +68,11 @@ private static async Task<bool> PossibleTypeArgumentAsync(Document document, Syn
                 // Case a?.M(x => x?.P)?.M2< : We need to analyze .M2
                 var innerMostConditionalAccessExpression = leftConditionalAccessExpression.GetInnerMostConditionalAccessExpression();
                 if (innerMostConditionalAccessExpression != null)
-                {
                     leftNode = innerMostConditionalAccessExpression.WhenNotNull;
-                }
             }
 
-            var info = model.GetSymbolInfo(leftNode, cancellationToken);
+            var semanticModel = await document.ReuseExistingSpeculativeModelAsync(leftNode.SpanStart, cancellationToken).ConfigureAwait(false);
```

Comment:
we use ReuseExistingSpeculativeModelAsync so we can share/benefit from speculative semantic models if available.  this shoudl often be the case as this is only for BinaryExpressions, which are normally in method bodies (or field initializers).

Reply:
I wonder if here this should be using the frozen document as well?  Or does that not matter when using `ReuseExistingSpeculativeModelAsync` (from the docs it looks like it uses some cached semantic model)